app/controllers/api/v1/statuses/bookmarks_controller.rb

```diff
@@ -17,8 +17,12 @@ def destroy
     @status = requested_status
     @bookmarks_map = { @status.id => false }
 
-    bookmark = Bookmark.find_by!(account: current_user.account, status: @status)
-    bookmark.destroy!
+    begin
+      bookmark = Bookmark.find_by!(account: current_user.account, status: @status)
```

Comment:
It is easier to just use `find_by` without exclamation point and `bookmark&.destroy!`